ios/Scripts/generate_app_icons_from_source.py

```
@@ -0,0 +1,111 @@
+        # iPad
+        "AppIcon-20@1x.png": 20,  # 20x20 @1x
+        "AppIcon-29@1x.png": 29,  # 29x29 @1x
+        "AppIcon-40@1x.png": 40,  # 40x40 @1x
+        "AppIcon-76@1x.png": 76,  # 76x76 @1x
+        "AppIcon-76@2x.png": 152,  # 76x76 @2x
+        "AppIcon-83.5@2x.png": 167,  # 83.5x83.5 @2x
+        # App Store
+        "AppIcon-1024.png": 1024,  # 1024x1024
+    }
+
+    # Путь к папке с иконками (относительно расположения скрипта)
```

Ruff (0.13.1)

Comment contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF003)